### ISSUE TITLE
Themata

### ISSUE THREAD

**Issue by IhateTrains** [loup99/WtWSMS#386]
This issue is supposed to contain the discussion and research about the Themata system.

**Comment by IhateTrains** [comment]
This is the earliest theme setup I have found so far: https://eaxelandersson.deviantart.com/art/Themes-of-the-Byzantine-Empire-in-the-7th-Century-563426508
I think the quite large size and the fact that they were directly under the emperor could speak for the early themes being kingdom-tier. What do you think?

**Comment by loup99** [comment]
![image](https://user-images.githubusercontent.com/7620180/41189629-2066c6ea-6bd1-11e8-9dde-8c3732b3d1a3.png)

This picture seems to be the original source of that, from Haldon, J. F. _Byzantium in the Seventh Century: The Transformation of a Culture_. Cambridge: Cambridge University Press, 1990. 

On the matter of scale kingdom seems preferable on this basis.

**Comment by IhateTrains** [comment]
I think there may be a problem with the chronology in the Greek official language and Themata duo.
The emperor responsible for introducing Greek as the Roman Empire's official language was Heraclius (reign 610 – 641), while modern historians date the creation of the themes to the period from the 640s to the 660s, under Constans II (r. 641 – 668).
More on this here: https://en.wikipedia.org/wiki/Theme_(Byzantine_district)#Origins

**Comment by loup99** [comment]
Loss of Egypt and Oriens -> Greek -> Themes 

The themes requirement was just to represent that there has been a loss.

**Comment by IhateTrains** [comment]
OK, I will try to write the conditions representing the loss of Oriens and Egypt for the Greek language decision.

**Comment by IhateTrains** [comment]
What I've got so far with the dynamic de-jure:
![2018-06-09](https://user-images.githubusercontent.com/29546927/41192917-f2a52640-6c05-11e8-9567-e8f67da50ca9.png)

**Comment by loup99** [comment]
Why are some in Grek and others in Latin? Is that a work in progress?

**Comment by IhateTrains** [comment]
These are all Greek names of the themes

**Comment by loup99** [comment]
First of all, Thrakesia is "Thrakesion" on Haldon's map. Secondly, aren't the other de jure names on the map mostly English or Anglicised Latin?

**Comment by IhateTrains** [comment]
Yes, it should Thrakesion.
Which thema names are English or Latin?

**Comment by loup99** [comment]
I'm talking about other titles, not themes. Like Thrace or Macedonia.

**Comment by IhateTrains** [comment]
They haven't been edited yet, but I can create themes for the Balkans.

**Comment by loup99** [comment]
Yes, but all titles across the map would need editing then. I'm not questioning the borders or titles, just the Greek names since no previous title uses Greek forms. Unless this represents the empire becoming more Greek?

**Comment by IhateTrains** [comment]
I think there's no problem with the Greek names, just like with the Roman names of Roman administrative titles. The Greek names are also the best to display (Anatolikon is much shorter than Anatolic Theme).

**Comment by IhateTrains** [comment]
What do you think about adding a new "Thematic" government, given to the ERE as part of the decision effect?

**Comment by loup99** [comment]
I'm opposed to it, it doesn't make sense from a realism point of view. Decentralisation and regionalisation is compatible with Bureaucratic.
